mlrun/api/utils/clients/sidecar/log_collector.py

```diff
@@ -12,21 +12,96 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-import mlrun.api.proto.log_collector_pb2
-import mlrun.api.proto.log_collector_pb2_grpc
+import mlrun.errors
```

Comment:
General comment on this file - I think `sidecar` is a very bad name for the folder. I would put this under `mlrun/api/utils/clients/log_collector/` since this is a log-collector client.

Reply:
change directory name to mlrun_services

Reply:
indeed sidecar is redundant, removed folder